apps/web/src/screens/Dashboard/Solana.tsx

```diff
@@ -56,6 +59,10 @@ export const SolanaDashboard: FC<Props> = () => {
 		appActions.showSendModal(Networks.solana);
 	};
 
+	const handleChangePrivateSetting = (next: boolean) => {
+		settingsActions.updatePrivateSettings({ _id: _id || '', isPrivate: next });
```

Comment:
like above, need another name for `isPrivate`

Reply:
I replaced `isPrivate` by `hideBalance`